Library/Formula/weechat.rb

```diff
@@ -14,6 +14,10 @@ class Weechat < Formula
 
   option "with-perl", "Build the perl module"
   option "with-ruby", "Build the ruby module"
+  option "with-python", "Build the python module"
```

Comment:
All of these, aside `tcl` are already implicitly stated below. The `optional` dependencies are essentially `option "with-python"` - Is there an upside/downside to adding more explicit options?